openjdk/excludes/ProblemList_openjdk19.txt

```diff
@@ -204,6 +204,12 @@ java/net/InetAddress/InternalNameServiceWithHostsFileTest.java https://github.co
 java/nio/channels/DatagramChannel/SendReceiveMaxSize.java https://github.com/adoptium/aqa-tests/issues/3820 aix-all
 java/net/DatagramSocket/SendReceiveMaxSize.java https://github.com/adoptium/aqa-tests/issues/3820 aix-all
 com/sun/net/httpserver/simpleserver/StressDirListings.java https://github.com/adoptium/aqa-tests/issues/3841 aix-all
+java/net/vthread/HttpALot.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+java/net/vthread/BlockingSocketOps.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+java/net/vthread/HttpALot.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+java/net/vthread/BlockingSocketOps.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+java/net/vthread/InterruptHttp.java https://github.com/adoptium/aqa-tests/issues/4024 aix-all
+jdk/net/ExtendedSocketOption/DontFragmentTest.java.DontFragmentTest https://github.com/adoptium/aqa-tests/issues/4024 aix-all
```

Comment:
DontFragmentTest.java is intermittently failing for a different reason than vthread unsupported, so we should not include it as part of this exclusion (which relates to #4024)
Execution failed: `main' threw exception: java.lang.UnsupportedOperationException: IPv6 not available

Reply:
@lumuchris256 - please remove line 212 from this PR

Reply:
noted